test/powershell/engine/ValidateAttributes.Tests.ps1

```diff
@@ -25,4 +25,111 @@ Describe 'Validate Attributes Tests' -Tags 'CI' {
             { function Local:foo { param([ValidateCount(2, 4)] [string[]] $bar) }; foo 1,2,3,4 } | Should Not Throw
         }
     }
+
+    Context "ValidatePositive" {
+        BeforeAll {
+           $testCases = @(
+               @{ sb = { function Local:foo { param([ValidatePositive()] [int] $bar) }; foo -1  }; FullyQualifiedErrorId = "ParameterArgumentValidationError,foo";  InnerErrorId = "ValidateRangeTooSmall" }
```

Comment:
I'll get this cleaned up. I was trying to follow the existing format of the file. Is it cool to update those other test as part of this PR as well?